tests/__snapshots__/PageLifecycleProvider.spec.tsx.snap

```diff
@@ -1,5 +1,38 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`PageLifecycleProvider Does not raise events twice for a single route 1`] = `
```

Comment:
If you check the commits in this PR, you will see the test replicated the issue, then the snapshot was updated to remove the duplicated event.